src/igvfd/audit/tissue.py

```diff
@@ -10,7 +10,11 @@
 
 @audit_checker('Tissue', frame='object')
 def audit_tissue_ccf_id(value, system):
-    '''Tissue objects must specify a common coordinate framework identifier (CCF ID) required for human data.'''
+    '''
+        audit_detail: Tissues must specify a common coordinate framework identifier (CCF ID) required for human data.
```

Comment:
tissue samples are expected to specify a common coordinate framework identifier (CCF ID).